doc/package-blueprint.yaml

```diff
@@ -0,0 +1,72 @@
+# Other properties than designer are not consolidated.
+# They are just plain as they are in Mashery
+package:
+  name: Some package
+  description: Package description
+  eav: {}
+  keyLength: 24
+  nearQuotaThreshold: 50
+  notifyAdminEmails: Mashery_Support@adidas.com
+  notifyAdminPeriod: day
+  notifyDeveloperPeriod: day
+
+# Plans are not nested under package to simplify structure
+# Plans are array of objects (hashes)
+plans:
+# First plan with all allowed properties
+- name: Plan name
+  description: This is an unlimited PLN demo
+  status: active
+  description: This is a plan which should be used by application's which needs to write into Documentum API.
+  emailTemplateSetId: null
+  eav: {}
+  notes: null
+  maxNumKeysAllowed: 4
+  numKeysBeforeReview: 0
+  qpsLimitCeiling: 1000
+  rateLimitCeiling: 0
+  rateLimitPeriod: day
+  adminKeyProvisioningEnabled: true
+  qpsLimitExempt: false
+  qpsLimitKeyOverrideAllowed: true
+  rateLimitExempt: true
+  rateLimitKeyOverrideAllowed: true
+  responseFilterOverrideAllowed: true
+  selfServiceKeyProvisioningEnabled: false
```

Comment:
Notification Template='KeyRegistrationTemplate'

Reply:
@rathonit1 
It is property `emailTemplateSetId`
It is adept to do the same thing as with organization where you can provide `id` or `name`.
Let's say we will rename it to `notificationTemplate` and value can be id or name and app will resolve it itself

Reply:
I am able to see now (after looking in to backup data), my bad.